Common/Sources/Extensions/UIViewController.swift

```diff
@@ -89,4 +89,14 @@ extension UIViewController: HyperwalletFlowDelegate {
             Holder.initializationData[ObjectIdentifier(self)] = newValue
         }
     }
+
+    /// Removes the current coordinator while moving back
+    public func removeCoordinator() {
+        Holder.coordinator.removeValue(forKey: ObjectIdentifier(self))
+    }
+
+    /// Removes all coordinators
+    public func removeAllCoordinators() {
+        Holder.coordinator.removeValue(forKey: ObjectIdentifier(self))
+    }
```

Comment:
Why is removeAllCoordinators required? Also it's just removing 1 coordinator and not all

Reply:
Updated to remove all coordinators. This can be called during logout.